gnd/src/main/res/layout/fragment_terms_service.xml

```diff
@@ -29,15 +29,33 @@
     android:layout_height="match_parent"
     tools:context=".ui.terms.TermsOfServiceFragment">
 
-    <androidx.appcompat.widget.Toolbar
-      android:id="@+id/toolbar"
-      style="@style/ToolbarTheme"
-      android:layout_width="0dp"
-      android:layout_height="wrap_content"
-      app:layout_constraintEnd_toEndOf="parent"
+    <!-- CardView is used to create the drop shadow effect -->
```

Comment:
Nit: Final "."

Reply:
updated in https://github.com/google/ground-android/pull/845